Examples/Physics_applications/laser_acceleration/analysis_1d_fluids_boosted.py

```diff
@@ -51,17 +53,22 @@
 # ODE Function
 def odefcn(phi, xi, kp, a0, c, tau, xi_0, lambda_laser):
     phi1, phi2 = phi
-    a_sq = a0**2 * np.exp(-2 * (xi - xi_0)**2 / (c**2 * tau**2))*np.sin(2*np.pi*(xi - xi_0)/lambda_laser)**2
+    a_sq = (
+        a0**2
+        * np.exp(-2 * (xi - xi_0) ** 2 / (c**2 * tau**2))
```

Comment:
It's a little unfortunate that the standard is inconsistent with powers:
```
(xi - xi_0) ** 2
```
but
```
c**2
```
Is there a way to have this be consistent, i.e., use `()**2` instead of `() ** 2`?

Reply:
I think here are the rules on powers: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-breaks-binary-operators